Solutions/Lookout/Parsers/LookoutEvents.txt

```diff
@@ -23,7 +24,8 @@ let LookoutEvents_view  = view () {
 		DetailsAttributeChanges=column_ifexists('details_attributeChanges_s', ''),
 		Type=column_ifexists('type_s', ''),
 		ID=column_ifexists('id_s', ''),
-		EventTime=column_ifexists('eventTime_t', ''),
+        EventStartTime=column_ifexists('eventTime_t', ''),
+        TimeGenerated=column_ifexists('eventTime_t', ''),
```

Comment:
@rajendrakhabiyagate6 , TimeGenerated is a standard column of LA that will have the date and time that the record was created by the data source. Please refer https://docs.microsoft.com/en-us/azure/azure-monitor/logs/log-standard-columns. Can you please give another name here instead of TimeGenerated to avoid confusion between the standard TimeGenerated and the extended column in this parser. Please let us know. Thanks.

Reply:
@v-jayakal As per your comment Used another name instead of TimeGenerated field. Please check and approve our PR asap.

Thanks !!

Reply:
@rajendrakhabiyagate6 - Thanks, I am approving and merging it now.

Reply:
@rajendrakhabiyagate6 - Can you please proceed with solution packaging and publishing? Thanks.